Jenkinsfile

```diff
@@ -1063,7 +1063,7 @@ pipeline {
                         }
                     }
                 } // stage('Fault inection testing on EL 8.8')
-                stage('Test RPMs on EL 8.6') {
+                stage('Test RPMs on EL 8.8') {
```

Comment:
The RPMs are already being tested on EL 8.8 in the Functional test stages.  The entire point of this stage is to test the RPMs compatibility with EL 8.6.

Reply:
There are no Build RPM stage for EL 8.6 or Leap 15.4 (e.g. https://build.hpdd.intel.com/job/daos-stack/job/daos/job/release%252F2.4/455/pipeline-graph/) and there are no 15.4 RPMs in the artifacts: https://build.hpdd.intel.com/job/daos-stack/job/daos/job/release%252F2.4/455/artifact/artifacts/leap15/